lib/linter/apply-disable-directives.js

```diff
@@ -277,7 +380,7 @@ function applyDirectives(options) {
             };
         });
 
-    return { problems, unusedDisableDirectives };
+    return { problems, unusedDirectives };
```

Comment:
We should also update `unusedDisableDirectives` -> `unusedDirectives` in jsdoc:

https://github.com/eslint/eslint/blob/b7ef2f34fe12b68a366e1b4bf5f64d7332c6e72e/lib/linter/apply-disable-directives.js#L209